rpcs3/Emu/Cell/lv2/sys_uart.cpp

```diff
@@ -1,35 +1,597 @@
 #include "stdafx.h"
 
+#include "Emu/Cell/PPUThread.h"
 #include "Emu/Cell/ErrorCodes.h"
+#include "Emu/Cell/lv2/sys_sync.h"
 
 #include "sys_uart.h"
 
 LOG_CHANNEL(sys_uart);
 
-error_code sys_uart_initialize()
+struct av_get_monitor_info_cmd : public ps3av_cmd
 {
-	sys_uart.todo("sys_uart_initialize()");
+	u16 get_size(vuart_av_thread &vuart, const void *pkt_buf) override
```

Comment:
For these ones, I just comment out the argument names:
```
(vuart_av_thread& /*vuart*/, const void* /*pkt_buf*/)
```
This saves vertical space if you have a lot of unused variables and is also immediately more obvious compared to static_cast<void>. If you prefer explicit declaration of unused, then a macro would be required for readability.

Reply:
Done